### ISSUE TITLE
Play verb sent over WaitHook not being reproduced

### ISSUE THREAD

**Issue by AFelix53** [jambonz/jambonz-feature-server#124]
Sending a play verb over as a response to a WaitHook does not cause the audio available on the url property to be reproduced. 

```
        {
          "loop": "forever",
          "verb": "play",
          "url": "https://domain.com/static/jingle.wav"
        }
```

The same URL is reproduced correctly when used on a play verb that is returned, for example, on the incoming call webhook.

Below are the logs taken from feature-server of a call with that behaviour.

[feature-server-logs.zip](https://github.com/jambonz/jambonz-feature-server/files/9012155/feature-server-logs.zip)

**Comment by davehorton** [comment]
"not reproduced"....do you mean it is played only once?  Or not played at all?

**Comment by AFelix53** [comment]
It is not played at all, I tested other values for the loop property with no success either.

**Comment by davehorton** [comment]
Here is the relevant error:
```
   "stack": "AssertionError [ERR_ASSERTION]: The expression evaluated to a falsy value:\n\n  assert(rootSpan)\n\n    at new CallSession (/opt/app/lib/session/call-session.js:55:5)\n    at new ConfirmCallSession (/opt/app/lib/session/confirm-call-session.js:12:5)\n    at Conference._playHook (/opt/app/lib/tasks/conference.js:545:27)\n    at runMicrotasks (<anonymous>)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)\n    at async /opt/app/lib/tasks/conference.js:229:27",
      "msg": "Conference:_doWait: failed retrieving waitHook for conf:97ebc833-83e2-44a0-b063-11da40ddefc8:09daaee6-583a-4a81-a975-302d74635123",
```

This was fixed in this [commit](https://github.com/jambonz/jambonz-feature-server/commit/5be5b6d05d1a6be9b8da9e8e8f7742a5a7e83901) which is available in v0.7.6-rc1
